<center><strong>Water management and genetic loci regulating arsenic and cadmium
content in rice grains<strong>

<center><strong>Gareth Norton<sup>a</sup></strong>, A. Travis<sup>a</sup>, A. Douglas<sup>a</sup> and A.H. Price<sup>a</sup>

<center><i><sup>a</sup> School of Biological Sceicnes, University of Aberdeen, UK</i>

<center><i>g.norton@abdn.ac.uk</i>

<p style=text-align:justify>Rice is a global staple crop, being the main calorific component of many
people living subsistence livelihoods. Rice can accumulate toxic
elements such as arsenic and cadmium, and for these elements especially,
the crop water management strongly affects uptake. While a number of
genes have been identify that regulate the accumulation of arsenic and
cadmium in rice grains, these genes have been through the use of mutant
studies, over expression or gene knockouts. While the identification of
these genes have been instrumental in upstanding the processes of
arsenic and cadmium accumulation in rice, there is still scope to
utilise natural variation for these traits in breeding programmes. This
study was developed to assess genotype by water management interaction,
identify quantitative trait loci (QTL) for arsenic and cadmium
accumulation and propose candidate genes for lowering grain arsenic and
cadmium.

<p style=text-align:justify>This study utilises the Bengal and Assam Aus Panel to conduct genome
wide association (GWA) mapping for grain arsenic and cadmium in shoots
and grains of grown over two years under continually flooded (CF) and
alternate wetting and drying (AWD). AWD significantly reduced grain
arsenic across all cultivars on average by 15.7% and 15.1% in year 1 and 2. AWD significantly increased grain cadmium across all cultivars on
average by 50.0% and 92.4% in year 1 and 2 respectively compared to the
plants grown under CF. There was strong cultivar by treatment
interaction for grain cadmium but it was weaker for arsenic. All traits
were strongly influenced by cultivar. GWA mapping identified a large
number of QTLs for both arsenic and cadmium traits, with respectively
six and- 26 QTLs showing stability across years and/or water treatments.
Six of the loci were investigated in detail using an approach of
clustering cultivars that had similar haplotypes for the QTL regions and
then looking at the phenotypic values across the clusters. A number of
the identified QTLs co-localised with known genes involved in arsenic
(Lsi2) and cadmium (OsNRAMP5) accumulation. This study has identified a
number of novel QTLs for arsenic and cadmium accumulation, as well as
cultivars that consistently accumulate less of these elements over
multiple field traits. The use of a haplotype clustering approach after
genome wide association mapping has allowed for the affect, in terms of
elemental accumulation, to be determined for cultivars that share
similar genomic sequence. These haplotypes can be further investigated
for their potential in breeding low arsenic and cadmium rice cultivars
by utilising natural variation.

