<center><strong>Microbial communities associated with mercury methylation in paddy soils

<center><strong>Yu-Rong Liu<sup>a,b*</sup></strong>, A. Johs<sup>c</sup>, L. Bi<sup>b</sup>, X. Lu<sup>c</sup>, J.Z. He<sup>b</sup> and
B.H. Gu<sup>c</sup>

<center><i><sup>a</sup> College of Resources and Environment, Huazhong Agricultural
University, China<i>

<center><i><sup>b</sup> Research Center for Eco-Environmental Sciences, Chinese Academy of
Sciences, China<i>

<center><i><sup>c</sup> Environmental Sciences Division, Oak Ridge National Laboratory,
United</i> States

<center><i>yrliu@mail.hzau.edu.cn<i>

<p style="text-align:justify">Mercury (Hg) is a growing environmental concern because it can be
converted to neurotoxic methylmercury (MeHg) that can bioaccumulate and
biomagnify up the food chain. The consumption of foods contaminated with
MeHg threatens human health globally. Seafood, especially large
predatory fish, is widely considered as the primary route for human
exposure to MeHg, but recent studies show that rice may constitute
another important exposure route, particularly in countries where rice
is a staple of the everyday diet. Although certain anaerobic
microorganisms are known to convert inorganic mercury (Hg) to MeHg and
the discovery of a two-gene cluster <i>hgcAB</i> has linked Hg methylation to
several phylogenetically diverse groups of microorganisms, the
prevalence and diversity of Hg methylators in microbial communities of
rice paddy soils remain unclear. Here, we report the abundance and
distribution of <i>hgcAB</i> genes using a combination of 3rd-generation
PacBio sequencing, metagenomic and quantitative PCR analyses in several
mine-impacted paddy soils from southwest China. We show that Hg
methylating communities are diverse, with iron-reducing bacteria (i.e.,
<i>Geobacter</i>) and methanogens as the dominant taxa likely involved in Hg
methylation in the soil. Phylogenetic analysis also uncovered some hgcAB
sequences closely related to three novel Hg methylators, <i>Geobacter</i>
<i>anodireducens</i>, <i>Desulfuromonas</i> sp. DDH964, and <i>Desulfovibrio</i> sp. J2
in these paddy soils. These findings shed new light on microbial
community composition and major clades driving Hg methylation in rice
paddy soils and have important implications in developing strategies for
minimizing bioaccumulation of MeHg in rice grains and thus human
exposure. Our findings also offer new insights into the biogeochemical
factors controlling MeHg production and have the potential global
implications for understanding the evolution and distribution of Hg(II)
methylating microorganisms not only in paddy habitats but also other
ecosystems such as wetlands.
