<center><strong>Arsenic methylation efficiency across microbial phyla </strong>

<center><strong>Karen Viacava<sup>a</sup></strong>, S. Dyer<sup>a</sup>, K. Lederballe Meibom<sup>a</sup>, A. Mestrot<sup>b</sup>
and R. Bernier-Latmani<sup>a</sup>

<center><i><sup>a</sup> Environmental Microbiology Laboratory, École Polytechnique Féderal de Lausanne, Switzerland.</i>

<center><i><sup>b</sup> Trace Element Speciation Group, University of Bern, Switzerland </i>

<center><i>karen.viacava@epfl.ch <i>

<p style="text-align:justify">Arsenic (As), one of the most toxic elements in the periodic table, is
naturally present in the environment where it undergoes extensive
microbial cycling. Because of the toxicity of this metalloid, resistance
mechanisms developed by Archaean ancestors to thrive under the presence
of reduced As during the earliest anoxic ecosystems have been preserved
and are present today in many bacteria and archaea genes. An arsenite
efflux pump system or the mechanisms for arsenate reduction or
oxidation, to name a few.

<p style="text-align:justify">Arsenic methylation has been proposed as another resistance mechanism
but also as an activation pathway or a precursor reaction for the
synthesis of more complex arseno-organic molecules. It is a
microbially-mediated process resulting in the addition of one or several
methyl groups to inorganic As. The reaction is catalyzed by the enzyme
arsenite methyltransferase (ArsM), generating volatile and non-volatile
arsenicals.

<p style="text-align:justify">Rice, main staple food for millions of people worldwide, particularly
efficient in accumulating arsenic represents one of the main sources of
arsenic-exposure for humans. Methylated arsenic, in certain regions
represents the predominant species present in rice. However, rice plants
are not able to methylate As and soil microorganisms are the responsible
mediators of this process.

<p style="text-align:justify">Several studies have been done to identify the main microbial drivers of
As methylation by <i>arsM</i> sequencing and yet very few microbes have been
confirmed as active As methylators by pure strains like the Bacteroidete
<i>Arsenicibacter rosenii</i> SM-1 and the Firmicute <i>Clostridium</i> <i>sp</i>. BXM.
Nonetheless, members of numerous other phyla present in soils harbor
this gene and it is unclear whether all represent functional proteins
and active methylating organisms. Thus, the goal of this study was to
systematically probe the functionality and the <i>in vivo</i> activity of
ArsM across different phyla in microorganisms isolated from or whose
<i>arsM</i> gene has been found in paddy soils.

<p style="text-align:justify">By performing time-course experiments, we assessed the As resistance and
the capacity for arsenite methylation and volatilization across seven
microbial strains encoding the <i>arsM</i> gene: two archaea (<i>Methanosarcina
mazei</i> Gö1, <i>Methanosarcina acetivorans</i> C2A); two Firmicutes
(<i>Anaeromusa acidaminophila</i> DSM 3853, <i>Clostridium pasteurianum</i> DSM
525); a Streptomycete (<i>Streptomyces vietnamensis</i> DSM 41927); a
Deltaproteobacterium (<i>Geobacter metallireducens</i> GS-15); and a
Bacteroidete (<i>Arsenicibacter rosenii</i> SM-1). In addition, all <i>arsM</i>
genes were cloned into the arsenic sensitive <i>Escherichia coli</i>
AW3110(DE3) and As speciation performed.

<p style="text-align:justify">The results show that most of the strains were not able to methylate As
despite harboring <i>arsM</i> genes that encode functional ArsM proteins. We
hypothesize that more efficient As detoxification pathways might be
prevalent, precluding methylation. We conclude that the presence of
<i>arsM</i> does not equate As methylation activity and more work is
undergoing to deconvolute <i>arsM</i> regulation.
