<center><strong>Monomethylmercury degradation by the human gut microbiota is
stimulated by protein amendments</strong>

<center><strong>Galen Guo,</strong> E. Yumvihoze, H.M. Chan and A.J. Poulain

<center><i>UniversityÂ of Ottawa, Department of Biology, Canada<i>

<center><i>Gguo094@uottawa.ca<i>

<p style="text-align:justify">Mercury (Hg) is considered by the World Health Organization as one of
the top ten chemicals of major public health concern. Its organic form,
monomethylmercury (MMHg), poses the highest risk as it can be
biomagnified in aquatic systems resulting in elevated exposure among
fish and rice consumers. The gut microbiome was identified as a
potential factor for the variations among individuals in Hg uptake.
However, little is known about the role of gut microbiota in the
gastrointestinal tract on Hg transformations and bioavailability.
Current literature has shown that the human gut microbiome can
demethylate MMHg, however, no known demethylating genes or microbial
guilds were identified. Our goal is to determine if the alteration of
the gut microbiome by diet amendment would enhance MMHg demethylation.
Using a series of batch experiments, we evaluated the effects of
changing nutritional component of the diet (by altering relative
abundances of carbohydrate or protein) in the gut microbial community
structure of two individuals. We tracked Hg methylation and
demethylation rates using stable isotope tracers and conducted high
throughput 16S rRNA amplicon sequencing to determine the microbial
community structure. We observed &gt; 90% decrease in initial MMHg
concentration in protein-rich diet mixed with the fecal slurry of
individual A; individual B gut microbiota did not exhibit such
demethylation trend. No methylation was observed in any treatment. We
followed up this experiment by mixing in different ratios of the fecal
microbiota of individual A and B. The MMHg demethylation phenotype was
conserved, suggesting individual A contains microbiota element necessary
for MMHg demethylation that individual B does not possess. Lastly, to
identify microbes involved in demethylation, we performed a series of
experiments using inhibitors of 2 known demethylation reactions (i.e.,
sulfate reduction, methanogenesis). None of the inhibitors tested
prevented demethylation indicating a new mechanism is involved. Our
results indicate that the gut microbiome of individual A contains a
microbiota required for demethylation of MeHg that individual B lacks. A
comparative metagenomic analysis between the different nutritional
amendment and individuals is underway. Our results are important in the
understanding of the mechanisms for MMHg demethylation in the human gut
and the variations in MMHg absorption and toxicity.
