<center><strong>Microbial-mediated antimony cycling revealed by DNA-SIP and
metagenomic-binning: case studies of advance molecular tools to unravel
the mystery of biogeochemical antimony cycling</strong>

<center><strong>Weimin Sun<sup>a</sup></strong>, M. Zhang<sup>a</sup>, M. M. Haggblom<sup>b</sup>, B. Li<sup>a</sup>, X. Sun<sup>a</sup>
and F. Li<sup>a</sup>

<center><i><sup>a</sup> Guangdong Institute of Eco-environmental Science & Technology,
China</i>

<center><i><sup>b</sup> Department of Biochemistry and Microbiology, Rutgers University,
United States</i>

<center><i>wmsun@soil.gd.cn</i>

<p style=text-align:justify>Biotransformation of antimony (Sb) is a newly discovered process
regulating the Sb redox transformation in soils. Understanding such
biogeochemical process may contribute to future bioremediation strategy
to remove Sb from environments. In comparison to As, the relative of Sb
belonging to the same group of periodic table, the biotransformation of
Sb is much less studies. Our current understanding of the Sb-oxidizing
and â€“reducing bacteria is entirely based on several pure isolates. In
this study, we employed the stable isotope probing coupled with the
metagenomic-binning to identify the microorganisms participating the Sb
oxidation and reduction and reveal their potential mechanism to
transform Sb. Sb-contaminated soils sampled from active Sb mining area
were selected. Metagenomic analysis on this soil indicated that
As-related genes (e.g., <i>arsC</i> and <i>aioA</i>) are correlated with Sb
species, suggesting that microorganisms may use similar pathways to
oxidize or reduce Sb as As. In order to verify this observation, DNA-SIP
was performed to identify the potential Sb oxidizers and reducers in the
microcosms inoculated from Sb-contaminated rice paddy soils and dry land
soils. DNA-SIP indicated that <i>Pseudomonas</i>, <i>Lysinibacillus</i>, and
<i>Geobacter</i> may be the potential Sb(V)-reducing bacteria. Further,
metagenomic-binning was performed to reveal the physiological traits of
the potential Sb(V)-reducing bacteria identified by DNA-SIP. Our data
indicated that the Sb(V)-reducing bacteria may use the respiratory
arsenate reductase (<i>arrABD</i>) to reduce Sb(V) to Sb(III). However,
members of <i>Lysinibacillus</i> may use other mechanisms to transform Sb(V)
instead of <i>arrABD</i>. Sb(III) oxidation also represents a very important
geochemical process, which can reduce the toxicity of Sb. Therefore,
experiments are currently on the way to identify the potential
autotrophic Sb(III)-oxidizing bacteria in microcosms seeded from
Sb-contaminated soils. Several novel Sb-metabolizing bacteria were
identified in this study, indicating that DNA-SIP is a powerful tool to
identify the Sb-metabolizing bacteria. The combination of DNA-SIP and
metagenomic-binning suggested the effectiveness of these advance
molecular techniques in unravelling the mystery of biotransformation of
Sb. This work represents the first application of DNA-SIP and
metagenomic-binning to identify Sb-metabolizing bacteria and their
potential metabolic potentials and therefore provides a valuable set of
data to definitively link identity with Sb cycling. The identification
of novel putative Sb-metabolizing bacteria expands current knowledge
regarding the Sb cycling.
